fhir-persistence-scout/pom.xml

```diff
@@ -18,6 +18,26 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
Bad Merge?

Reply:
this is actually here on purpose, but its only tangentially related to the rest of the PR.
basically, I was trying to use liberty dev mode in vscode.  without this plugin, our experimental fhir-persistence-scout project fails to compile in vscode (because the classes in there depend on this one generated class that winds up in generated-sources).  the config here tells vscode to add the folder as a source folder.

Reply:
see https://github.com/redhat-developer/vscode-java/issues/177 for a long sorted history..